### PR TITLE
Add support for onlyIfResumed argument in navigateBack function inside ResultBackNavigator

### PR DESCRIPTION
**Pull request by petrstetka**
Add support for #180 enhancement

### CODE REVIEW

File: compose-destinations/src/main/java/com/ramcosta/composedestinations/result/ResultBackNavigator.kt [hunk 1]
```diff
@@ -32,8 +32,15 @@ interface ResultBackNavigator<R> {
...
+     *
+     * @param onlyIfResumed if true, will ignore the navigation action if the current `NavBackStackEntry`
+     * is not in the RESUMED state. This avoids duplicate navigation actions.
+     * By default is false to have the same behaviour as [NavController].
      */
-    fun navigateBack(result: R)
+    fun navigateBack(
+        result: R,
+        onlyIfResumed: Boolean = false
+    )
```
Comment (raamcosta) [inline compose-destinations/src/main/java/com/ramcosta/composedestinations/result/ResultBackNavigator.kt:43]: I believe, if we do this, it also makes sense to do it in the `navigateBack` with no result. What do you think?

**Comment by raamcosta** [comment]
> Add support for #180 enhancement

Thanks a lot @petrstetka! Looks great 🙂 
I'm just considering a small change, as said in the other comment.

File: compose-destinations/src/main/java/com/ramcosta/composedestinations/result/ResultBackNavigator.kt (hunk 1 above)
Comment (petrstetka) [reply compose-destinations/src/main/java/com/ramcosta/composedestinations/result/ResultBackNavigator.kt:43]: Sure, I added there also :)

**Comment by petrstetka** [comment]
Hello @raamcosta, I added `onlyIfResumed` argument also to `navigateBack` with no result. Is it fine like this now?

**Review by raamcosta — APPROVED** [review]

**Comment by raamcosta** [comment]
Thanks again @petrstetka !

**Pull request merged**